libs/angular-accelerator/src/lib/components/content/content.component.html

```diff
@@ -0,0 +1,3 @@
+<div [ocxContent]="title" [style]="styleClass">
```

Comment:
The `styleClass` input should be applied as a class, not as a custom style. The same also applies to the content container component. Please also make sure that the style classes provided via input have a higher priority than the classes which are added by the directive itself. This way we don't only allow devs to specify additional classes but also allow them to overwrite existing style classes such as padding through the given input.

Reply:
Please verify the class merging behavior described above in a unit test.